### PR TITLE
Missing register and test

### PR DESCRIPTION
**Pull request by roigcarlo**
Hotfix and test.

### CODE REVIEW

**Review by loumalouomega — DISMISSED** [review]
Thanks

File: kratos/tests/cpp_tests/global_pointers/test_global_pointers_vector.cpp [hunk 1]
```diff
@@ -46,8 +47,38 @@ KRATOS_TEST_CASE_IN_SUITE(GlobalPointersContainerTest, KratosCoreFastSuite)
...
+}
+
+KRATOS_TEST_CASE_IN_SUITE(GlobalPointersContainerInVariableTest, KratosCoreFastSuite)
+{
+    Model current_model;
+    Model loaded_model;
+    ModelPart& mp = current_model.CreateModelPart("test");
+    mp.AddNodalSolutionStepVariable(TEMPERATURE); //not to have an empty var list
+
+    const auto & node_1 = mp.CreateNewNode(1,1.0,2.0,3.0);
```
Comment (loumalouomega) [inline kratos/tests/cpp_tests/global_pointers/test_global_pointers_vector.cpp]: Put the & in the middle is the most unholy thing
imagen

Comment (roigcarlo) [reply kratos/tests/cpp_tests/global_pointers/test_global_pointers_vector.cpp]: Weeeeeell I've move it... (But I still prefer to put it in the middle when passing arguments)

**Comment by roigcarlo** [comment]
@loumalouomega approve again pls ?

**Comment by loumalouomega** [comment]
> @loumalouomega approve again pls ?

imagen

**Review by loumalouomega — APPROVED** [review]

**Pull request merged**